setup.py

```diff
@@ -64,6 +64,7 @@
         "Sphinx~=4.3.2",
         "wrapt~=1.13.3",
         "pyzmq~=22.3.0",
+        "uvloop~=0.17.0",
```

Comment:
You should add the requirement in the same commit as the code where you use it. That way, figuring out why the requirement was added is much easier.

Reply:
Fixed.